tests/Feature/FlightControllerTest.php

```diff
@@ -0,0 +1,403 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Airline;
+use App\Models\City;
+use App\Models\Flight;
+use Database\Seeders\AirlineSeeder;
+use Database\Seeders\CitySeeder;
+use Database\Seeders\DatabaseSeeder;
+use Database\Seeders\FlightSeeder;
+use DateTime;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use PHPUnit\Framework\Attributes\DataProvider;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Date;
+use Tests\TestCase;
+
+class FlightControllerTest extends TestCase
+{
+    use RefreshDatabase;
+    private $baseUrl = '/api/flights';
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed([DatabaseSeeder::class]);
+    }
+
+    /** @test */
+    public function it_should_return_a_list_of_flights(): void
+    {
```

Comment:
```suggestion
        $this
            ->getJson($this->baseUrl)
            ->assertOk() // Equals to assertStatus(200)
            ->assertJsonStructure([
                'data' => [
                    '*' => [
                        'id',
                        'airline',
                        'origin_city',
                        'destination_city',
                        'departure_at',
                        'arrival_at'
                    ],
                ],
            ]);
```
You can chain assertions (applies to this and the rest of tests). Are you certain the response is paginated as you expect (Amount of records per page, current page, etc)?